src/app.css

```diff
@@ -3,10 +3,23 @@
 @plugin '@tailwindcss/typography';
 
 body {
+	position: relative;
```

Comment:
whenever possible (which should be pretty close to 100% of the time), please use tailwind `@apply` directives instead of CSS styles: https://tailwindcss.com/docs/functions-and-directives#apply-directive